packages/react-server-data-bundle-cache/src/wrap.js

```diff
@@ -0,0 +1,25 @@
+import _ from "lodash";
```

Comment:
Yep.  Only if you use the plugin.

And it would be anyway by `stratocacher`, so I figured I might as well use it here, too.